packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -121,13 +121,30 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
 
     const { isStickyTop, isStickyBottom } = this.state;
 
+    const nonStickyContent =
+      this._nonStickyContent && this._nonStickyContent.current
+        ? {
+            offsetHeight: this._nonStickyContent.current.offsetHeight,
+            scrollWidth: this._nonStickyContent.current.scrollWidth
+          }
+        : undefined;
+    const placeHolder =
+      this._placeHolder && this._placeHolder.current
+        ? {
+            offsetHeight: this._placeHolder.current.offsetHeight,
+            scrollWidth: this._placeHolder.current.scrollWidth
+          }
+        : undefined;
+
```

Comment:
Let's have two helper method instead of one, similar to `isHeightOrWidthDifferent` - `isOffsetHeightDifferent `& `isScrollWidthDifferent`. And use that in the expression below.